profitbricks/src/test/java/org/jclouds/profitbricks/binder/drive/AddRomDriveToServerRequestBinderTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.profitbricks.binder.drive;
+
+import org.jclouds.profitbricks.domain.Drive;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import org.testng.annotations.Test;
+
+@Test(groups = "unit", testName = "AddRomDriveToServerRequestBinderTest")
+public class AddRomDriveToServerRequestBinderTest {
+
+   @Test
+   public void testAddRomDriveToServerPayload() {
+      AddRomDriveToServerRequestBinder binder = new AddRomDriveToServerRequestBinder();
+
+      Drive.Request.AddRomDriveToServerPayload payload = Drive.Request.AddRomDriveToServerPayload.builder()
+              .serverId("server-id")
+              .storageId("image-id")
+              .deviceNumber("device-number")
+              .build();
+
+      String actual = binder.createPayload(payload);
+      assertNotNull(actual, "Binder returned null payload");
```

Comment:
nit: This is redundant, the next check will also fail as the expected payload is not null.